config/config.global.js

```diff
@@ -0,0 +1,6 @@
+var config = module.exports = {}
+
+config.env = 'development'
+
+config.discordjs = {}
```

Comment:
The token itself isn't specific to discord.js, so `discord`. Might be switching away from discord.js I think.